go/pkg/client/tree.go

```diff
@@ -131,9 +132,36 @@ func getTargetRelPath(execRoot, path string, symMeta *filemetadata.SymlinkMetada
 	return relExecRoot, relSymlinkDir, err
 }
 
+// getRemotePath returns a remote path by transforming the local input path
+// in the transformed path workingDir component is replaced with remoteWorkingDir
+func getRemotePath(path, workingDir, remoteWorkingDir string) (string, error) {
+	workingDirRelPath, err := filepath.Rel(workingDir, path)
+	if err != nil {
+		return "", fmt.Errorf("getRemotePath failed while trying to get working dir relative path of %q, err: %v", path, err)
+	}
+	remotePath := filepath.Join(remoteWorkingDir, workingDirRelPath)
+	return remotePath, nil
+}
+
+// getNormalizedPaths returns local and remote normalized paths for a given local absolute path
```

Comment:
What is "normalized" in this context, can you clarify please?

Reply:
good point. it made more sense within `loadFiles` before extracting it to a separate function but it's not clear now after the context from `loadFiles` is missing. I've renamed it to `getExecRootRelPaths` to clarify the function's purpose